js/controller.js

```diff
@@ -578,8 +574,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onBitrateInfoAvailable: function(event, bitrates) {
-      this.state.videoQualityOptions.availableBitrates = bitrates;
-      this.renderSkin({"videoQualityOptions": {"availableBitrates": bitrates}});
+      if (bitrates && bitrates.bitrates) {
+        this.state.videoQualityOptions.availableBitrates = bitrates.bitrates;
+        this.renderSkin({"videoQualityOptions": {"availableBitrates": bitrates.bitrates }});
+      }
```

Comment:
if PBF is going to pass the bitrates in the same format, then it looks fine to me.
@nagavenivn, can you look at this?